model/import/TemplateInterface.php

```diff
@@ -28,4 +28,6 @@ interface TemplateInterface
     public function getId(): string;
 
     public function getDefinition(): array;
+
+    public function getQtiTemplate(): string;
```

Comment:
As we discussed, this should not be here, now we are tying up Template with QTI. Please, lets move this to another template or add this as part of the definition array instead.